apps/nextjs-website/react-components/components/common/ContainerRC.tsx

```diff
@@ -14,6 +14,7 @@ interface ContainerProps {
   px?: BoxProps['px'];
   spacing?: GridProps['spacing'];
   sx?: GridProps['sx'];
+  maxWidth?: 'lg' | 'xl';
```

Comment:
```suggestion
  size?: 'lg' | 'xl';
```
I think `size` is a clearer name.
`maxWidth` makes me think I can input a number.